license_check.py

```diff
@@ -81,7 +82,7 @@ def check_license(filename, spdx_expr):
 
                 return True
 
-    eprint(f'License not found in {filename}')
+    eprint(f'License not found in ...\t\t{filename}')
```

Comment:
```suggestion
    eprint(f'License not found: \t\t{filename}')
```

Reply:
Changed.

Reply:
@josecm Kindly resolve this conversation if the changes attend to your comments. Thank you.